platform/src/main/java/org/stellar/anchor/platform/utils/TrustAllSslEngineFactory.java

```diff
@@ -0,0 +1,79 @@
+package org.stellar.anchor.platform.utils;
+
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+import java.util.Map;
+import java.util.Set;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import org.apache.kafka.common.security.auth.SslEngineFactory;
+
+public class TrustAllSslEngineFactory implements SslEngineFactory {
```

Comment:
Is this what Anchors are expected to run in production? Should we trust all certificates?

Reply:
I thought about this. By not trusting all certificate, we need to configure `keystore` and `truststore`, mount the stores, configure the passwords, etc. It will make the configuration very difficult.

If there is a request from the anchors, we may add the configurations to read the CA certs from the file.

Reply:
This is a security risk, right? I think we may want to trust all certificates in our tests, but defaulting to trust all certificates seems incorrect to me. I see that we have the Keystore and the cert will be checked in as part of this PR, if we are not using them, should we delete them?

Reply:
We can delete those files. But they are good resources if we want to do the kafka auth tests in the `extended-tests`.

Reply:
It will be quite hard to provide the stores and pass the passwords through the secret env vars. 

In most cases, the client and the kafka server are running in the same subnet and I think we can avoid the hassel of the keytore configurations.

Reply:
Don't we just need to pass the keystone and truststore locations in the properties?

Reply:
We will enable the certificate verification after discussion.